prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl

```diff
@@ -78,6 +78,12 @@ inputs = {
 
     #SLOT_TASK_HUBNAME = "ProductionTaskHub"
 
+    # TODO
+    IN_MEMORY_CACHE_TTL = 300
```

Comment:
@AleDore were you evaluating which is the best configuration here? 300 it's just a value I put base on my common sense.

Reply:
It should be 1 hour expressed in milliseconds as ADB2C JWT's expire time.